.gitignore

```diff
@@ -11,7 +11,7 @@ coverage.json
 typechain
 
 # don't push the environment vars!
-.env
+*.env
 
 # These environment variables are encrypted, but should not be pushed unless a secure password is used
 .env.enc
```

Comment:
You can remove this since you match above with *.env

Reply:
@KuphJr i added the * because currently its a hard match on .env only.  My notes.env file was getting staged